kernel/Makefile

```diff
@@ -172,13 +172,12 @@ endef
 # Build Targets
 # Debug targets only for latest stable and LTS stable
 #
-$(eval $(call kernel,4.14,4.14.x,$(EXTRA)))
-$(eval $(call kernel,4.14,4.14.x,-dbg))
-$(eval $(call kernel,4.13.13,4.13.x,$(EXTRA)))
-$(eval $(call kernel,4.13.13,4.13.x,-dbg))
-$(eval $(call kernel,4.9.62,4.9.x,$(EXTRA)))
-$(eval $(call kernel,4.9.62,4.9.x,-dbg))
-$(eval $(call kernel,4.4.98,4.4.x,$(EXTRA)))
+$(eval $(call kernel,4.14.1,4.14.x,$(EXTRA)))
+$(eval $(call kernel,4.14.1,4.14.x,-dbg))
+$(eval $(call kernel,4.13.15,4.13.x,$(EXTRA)))
```

Comment:
Did you intend to drop the 4.13 dgb kernel?

Reply:
yes as per commit https://github.com/linuxkit/linuxkit/pull/2767/commits/717829ea89b9ad1f10fad3d6c32a1364518557ac

Reply:
thanks, I only looked at the PR description!

Reply:
yeah, I should have mentioned it there but forgot...too many changes